_workinggroups/pyhep.md

```diff
@@ -75,3 +105,9 @@ Jeff Templon - Nikhef (Co-chair)
 The event was kindly sponsored by
 
 ![Nikhef](/images/pyhep/Nikhef_logo.png){:height="60px"} ![PSF](/images/pyhep/PSF_logo.png){:height="60px"}
+
+
+# Previous conveners
+
+2019 - Chris Tunnell (XENON1T)
```

Comment:
Tiny point - this formats all on a single line, so you might prefer to make it a bulleted list

Reply:
Oops, I missed that. Indeed the idea was to itemise, to read well. Will correct in a sec.

Thanks!